component.json

```diff
@@ -13,35 +13,58 @@
     }
   },
   "actions": {
-    "insert" : {
-      "title": "INSERT/UPDATE/DELETE",
-      "main": "./lib/actions/insert.js",
+    "insertBulk" : {
+      "title": "INSERT Bulk",
+      "main": "./lib/actions/insertBulk.js",
+      "description": "Insert multiple rows specified in incoming message in defined columns in a single table",
       "fields": {
-        "query": {
-          "label": "SQL Query",
+        "tableName": {
+          "label": "Table Name",
           "required": true,
-          "viewClass": "TextAreaWithNoteView",
-          "placeholder": "INSERT INTO films (code,title,kind) VALUES (${code},${title},${kind})",
-          "note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete"
+          "viewClass": "TextFieldView",
+          "placeholder": "schema.tableName",
+          "note": "Enter the name of the table into which the values will be inserted"
+        },
+        "columns": {
+          "label": "Columns",
+          "required": true,
+          "viewClass": "TextAreaView",
+          "placeholder": "column1, column2, column3",
+          "note": "Enter the name of the columns into which the values will be inserted"
         }
+      },
+      "metadata": {
+        "in": {
+          "type": "object",
+          "properties": {
+            "values": {
+              "type": "array",
+              "required": true,
+              "title": "Values"
+            }
+          }
+        },
+        "out": {}
       }
     },
-    "insertBulk" : {
-      "title": "INSERT bulk",
-      "main": "./lib/actions/insertBulk.js",
+    "insert" : {
+      "title": "INSERT/UPDATE/DELETE",
+      "main": "./lib/actions/insert.js",
+      "description": "INSERT or UPDATE or DELETE SQL Query",
       "fields": {
         "query": {
           "label": "SQL Query",
           "required": true,
           "viewClass": "TextAreaWithNoteView",
           "placeholder": "INSERT INTO films (code,title,kind) VALUES (${code},${title},${kind})",
-          "note": "You need use properties of message body as <i>${values}</i> in your  insert query"
+          "note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete"
```

Comment:
"note": "You can use properties of message body as <i>${values}</i> in your insert or update or delete query"